tasks/message/message.go

```diff
@@ -321,7 +330,17 @@ func parseMsg(m *messagemodel.Message, ts *types.TipSet, destCode string) (*mess
 	if !ok {
 		actor = statediff.LotusTypeUnknown
 	}
-	params, name, err := statediff.ParseParams(m.Params, int(m.Method), actor)
+	var params ipld.Node
+	var name string
+	var err error
+	go func() {
+		defer func() {
+			if r := recover(); r != nil {
+				err = fmt.Errorf("%v", r)
+			}
+		}()
+		params, name, err = statediff.ParseParams(m.Params, int(m.Method), actor)
```

Comment:
Is git diff being weird or is this (incorrectly) called in the anonymous go routine?

Reply:
great catch! fixed.